core/conversion/converters/impl/conv_deconv.cpp

```diff
@@ -29,36 +28,41 @@ auto conv_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns().patter
       auto out_padding = util::toDims(args[7].unwrapToIntList());
       LOG_DEBUG("out_padding: " << out_padding);
       int64_t groups = args[8].unwrapToInt();
+      LOG_DEBUG("groups: " << groups);
 
       nvinfer1::ILayer* new_layer;
       if (transposed) {
-        nvinfer1::IDeconvolutionLayer* deconv;
+        Weights bias;
         if (args[2].IValue()->isTensor()) {
-          Weights b(ctx, args[2].IValue()->toTensor());
-          deconv = ctx->net->addDeconvolutionNd(*in, w.num_input_maps, w.kernel_shape, w.data, b.data);
+          bias = Weights(ctx, args[2].unwrapToTensor());
         } else {
-          deconv = ctx->net->addDeconvolutionNd(*in, w.num_input_maps, w.kernel_shape, w.data, {});
+          bias = Weights(ctx, torch::zeros(args[1].unwrapToTensor().sizes()[1] * groups));
         }
 
+        auto deconv = ctx->net->addDeconvolutionNd(*in, w.num_input_maps * groups, w.kernel_shape, w.data, bias.data);
```

Comment:
Can you explain a bit about w.num_input_maps * groups and maybe mention it as a comment ?

Reply:
updated